DateTime.js

```diff
@@ -169,6 +169,10 @@ var Datetime = React.createClass({
 				update.open = this.state.open;
 			}
 		}
+        
+        if ( nextProps.viewMode !== this.props.viewMode ) {
```

Comment:
Sorry @simeg I'm not going to do anymore with the whitespace.  I fixed the issue at hand, so you can take it or leave it.